### PR TITLE
Update module to be able to integrate with landing-zone 

### PR DESCRIPTION
**Pull request by evpirnia**
Notable changes:
- Allow for deployment of logging and/or monitoring

### CODE REVIEW

File: workerpool.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+resource "ibm_container_worker_pool" "logmon" {
```
Comment (argeiger) [inline workerpool.tf]: Using the classic resource.  Please use https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/container_vpc_worker_pool

For subnets, you can get that from https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/container_cluster

Comment (argeiger) [inline workerpool.tf]: Using the classic resource.  Please use https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/container_vpc_worker_pool

For subnets, you can get that from https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/container_cluster

File: workerpool.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  worker_pool_name = "logmon-worker-pool"
+  machine_type     = "mx2.4x32" # 4 vCPU, 32 GB Memory
+  cluster          = var.cluster_name
+  size_per_zone    = 1
+  hardware         = "shared"
+  disk_encryption  = "true"
+  entitlement      = "cloud_pak"
+
+  # Ensure only the logging stack runs on this worker pool
+  taints {
```
Comment (argeiger) [inline workerpool.tf]: In our doc, the CR is using tolerations which isn't used above.  I am wondering if the CR has the taint and the node doesn't, what happens.  

If we taint the worker pool, we will need to have the CR with tolerations on it

Comment (argeiger) [inline workerpool.tf]: In our doc, the CR is using tolerations which isn't used above.  I am wondering if the CR has the taint and the node doesn't, what happens.  

If we taint the worker pool, we will need to have the CR with tolerations on it

**Comment by evpirnia** [comment]
@argeiger Added requested changes. Please review.

**Review by argeiger — APPROVED** [review]

File: workerpool.tf (hunk 1 above)
Comment (evpirnia) [reply workerpool.tf]: Resolving, worker pool creation is being handled by landing-zone

Comment (evpirnia) [reply workerpool.tf]: Resolving, worker pool creation is being handled by landing-zone

File: workerpool.tf (hunk 2 above)
Comment (evpirnia) [reply workerpool.tf]: Resolving, if node doesn't have taints it will accept every pods, no matter what tolerations will be defined in them.

Comment (evpirnia) [reply workerpool.tf]: Resolving, if node doesn't have taints it will accept every pods, no matter what tolerations will be defined in them.

**Pull request merged**